policies/versioning-policy.md

```diff
@@ -0,0 +1,155 @@
+  or functions are added in a feature release of the extant LTS release
+  and all supported intermediate successor releases.
+
+Exceptions to these rules require a vote by the [OMC].  See also the
+[stable release update policy].
+
+## Minor release: changing the second digit
+
+A minor release can, and generally will, introduce new features.  However both
+the API and ABI will be preserved.
+
+> For example, a program built with OpenSSL release 3.0.1 will be able to
+> run with OpenSSL 3.1.0 but might not be able to take advantage of new
+> features without modification.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Patch release: changing the third digit
+
+A patch release will only contain bug fixes.  Both the API and ABI will remain
+compatible across patch releases.
+
+> For example, a program linked with OpenSSL release 3.0.0 can
+> run with OpenSSL 3.0.1 without changes.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Long term stable releases
+
+A release can be designated as a release.  LTS releases will be supported
+for at least five years and the project will specify a LTS release at
+least every four years.  Non-LTS releases will be supported for at least
+two years.
+
+During the final year of support, we do not commit to anything other
+than security fixes.  Before that, bug and security fixes will be applied
+as appropriate.
+
+The addition of new platforms to LTS branches is acceptable so long as
+the required changes consist solely of additions to configuration.
+
+## Prerelease versions
+
+Before a major release the project will generally make a number of pre-releases.
+These are labelled _alpha_ and _beta_ releases.
+
+### Alpha releases
+
+An _alpha_ release is one that is:
+
+- not necessarily feature complete and
+- not necessarily includes all new APIs.
+
+### Beta releases
+
+A _beta_ release is one that:
+
+- is feature complete;
+- indicates a feature freeze and
+- only bug fixes are permitted.
+
+## Release criteria
+
+For a major or minor release, the following release criteria apply:
+
+- all open GitHub issues and pull requests older than 2 weeks at the time of
+  release need to be assessed for relevance to the version being released.
+  Any flagged with the a milestone for the version to be released must
+  be closed (see below);
+- clean builds in GitHub Actions for two days and
+- no open Coverity issues (not flagged as _False Positive_ or _Ignore_).
+
+Valid reasons for closing an issue/PR with a milestone for the version
+include:
+
+- we have just now or sometime in the past fixed the issue;
+- unable to reproduce (following discussion with original reporter
+  if possible);
+- working as intended;
+- deliberate decision not to fix this issue until a later release (this
+  wouldn't actually close the issue/PR but change the milestone instead) and
+- not enough information and unable to contact reporter.
+
+Exceptions require a vote by the [OTC].
```

Comment:
These seem to be OTC rules? Should this be an OTC policy?

Reply:
I think the entire release criteria section should be OTC.
The pre-release versions section seem like it should be too.

Reply:
In which case lets remove that section from this document.

Reply:
Done.  I left the pre-release definitions here.